lib/arclight/traject/ead2_config.rb

```diff
@@ -80,6 +80,19 @@
 
 to_field 'geogname_sim', extract_xpath('//xmlns:archdesc/xmlns:controlaccess/xmlns:geogname')
 
+# Indexes only specified controlled terms for archival description into the access_subject field
+to_field 'access_subjects_ssim', extract_xpath('//xmlns:archdesc/xmlns:controlaccess', to_text: false) do |_record, accumulator|
+  accumulator.map! do |element|
+    %w[subject function occupation genreform].map do |selector|
```

Comment:
Apologies for the unnecessary elements in https://github.com/projectblacklight/arclight/pull/585, I wasn't certain if it was desireable to have certain elements (e. g. `corpname`) indexed twice for a document.

Reply:
Oh its all good. I ran into this while looking at something else. Its confusing to me, but working through this, I think I understand now what we are after.